src/classes/OPP_OpportunityContactRoles_TEST.cls

```diff
@@ -45,9 +45,11 @@ private class OPP_OpportunityContactRoles_TEST {
     /*******************************************************************************************************
     * @description Role values to use in settings.
     */    
-    private static string donorRoleforTest = 'Donor';
-    private static string HonoreeRoleforTest = 'Honoree';
-    private static string NotificationRecipientRoleforTest = 'Notification Recipient';
+    private static String donorRoleforTest = 'Donor';
+    private static String HonoreeRoleforTest = 'Honoree';
+    private static String NotificationRecipientRoleforTest = 'Notification Recipient';
+    private static final String OPPORTUNITY_CONTACT_ROLE_INFLUENCER = 'Influencer';
+
```

Comment:
It would have been nice to either stay consistent with the other statics in the file, or convert them all to a similar uppercase format.  kinda hate seeing weird mixes within a class.

Reply:
Agreed. I'll update the other statics in the file.